indico/modules/auth/controllers.py

```diff
@@ -117,18 +119,32 @@ def _process(self):
         return multipass.logout(request.args.get('next') or url_for('misc.index'), clear_session=True)
 
 
-def _send_confirmation(email, salt, endpoint, template, template_args=None, url_args=None, data=None):
+def _send_confirmation(email, salt, endpoint, template, template_args=None, url_args=None, data=None, flashed_msg=None):
     template_args = template_args or {}
     url_args = url_args or {}
     token = secure_serializer.dumps(data or email, salt=salt)
     url = url_for(endpoint, token=token, _external=True, _secure=True, **url_args)
     template_module = get_template_module(template, email=email, url=url, **template_args)
     GenericMailer.send(make_email(email, template=template_module))
-    flash(_('We have sent you a verification email. Please check your mailbox within the next hour and open '
-            'the link in that email.'))
+
+    if not flashed_msg:
+        flash(_('We have sent you a verification email. Please check your mailbox within the next hour and open '
+                'the link in that email.'))
+    else:
+        flash(flashed_msg)
+
     return redirect(url_for(endpoint, **url_args))
 
 
+def _send_approval_mail(email):
+    url = url_for('.register', approval_token=secure_serializer.dumps(email, salt='approve-email-salt'), _external=True)
+    admin = db.session.query(User).filter_by(is_admin=True).first()
```

Comment:
First of all, you'd use `User.find_first(is_admin=True)` or `User.query.filter_by(is_admin=True).first()` (the latter is more flask-sqlalchemy-style, the former more indico-style) instead of `db.session.query`

However, sending an email to the "first" admin is a terrible idea. Email all admins or add a new settings on who to notify about new registrations. I'd go for the setting since not all admins might care about it.